app/assets/stylesheets/open_project_global/_variables.sass

```diff
@@ -34,7 +34,7 @@ $light-gray: #ccc
 
 $body-background:         white !default
 $body-font-family:        'Lato', 'Lucida Grande', Helvetica, Arial, sans-serif !default
-$body-font-color:         #555 !default
+$body-font-color:         #333 !default
```

Comment:
It was for the error messages: http://leaverou.github.io/contrast-ratio/#%23444-on-%23fedada

The old `#555` did not get a 7:1 contrast, that is why we went for `#444`.

Now that your default is `#333` the extra rule is not needed anymore...